lib/tasks/assets.rake

```diff
@@ -229,16 +238,21 @@ def copy_ember_cli_assets
   assets = {}
   files = {}
 
+  start = start_clock
   unless system("yarn --cwd #{ember_dir} install")
     STDERR.puts "Error running yarn install"
     exit 1
   end
+  log_task_duration('yarn install', start)
```

Comment:
Instead of having to start the clock manually and track it in a variable, what do you think about passing a block to the method? 

```
log_task_duration('yarn install') { # do something }
```